keyboards/jopr/readme.md

```diff
@@ -0,0 +1,6 @@
+jopr-106-Nl2SR-Cl2nL
```

Comment:
Aligning this template more with the [AVR keyboard template readme](https://raw.githubusercontent.com/qmk/qmk_firmware/c6850bad74ccec3cec0af1c74eb20b139d0ca481/quantum/template/avr/readme.md) would be useful to users as a primer, should they discover this keyboard through the QMK repository.

The link to your repo should still be included should the user want to read me.